src/app/components/Metadata/utils/index.tsx

```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
```

Comment:
May be able to remove `/** @jsx jsx */` and the `jsx` import from Emotion and just use `import React from 'react'`. Don't think we need theme access here.

Reply:
Oops, I copied this syntax from the Metadata component, which did use Theming 🤦‍♀️ . I'll update this now.